Makefile

```diff
@@ -261,10 +261,12 @@ clean: clean-certs clean-containers clean-slice ## Stops all containers and remo
 	$(RM) *.box
 
 test: ## executes the test script on a master
+	# This command is broken, as per
```

Comment:
It WAS moved, but it's not broken. You have to configure it: https://github.com/dcos/dcos#integration-tests